manager-parent/manager-v2/src/test/java/com/redhat/service/smartevents/manager/v2/api/user/ErrorsAPIV2Test.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.service.smartevents.manager.v2.api.user;
+
+import java.util.Collection;
+
+import com.redhat.service.smartevents.test.resource.PostgresResource;
+import io.quarkus.test.common.QuarkusTestResource;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.redhat.service.smartevents.infra.core.models.responses.ErrorListResponse;
+import com.redhat.service.smartevents.infra.core.models.responses.ErrorResponse;
+import com.redhat.service.smartevents.infra.v2.api.V2APIConstants;
+import com.redhat.service.smartevents.test.exceptions.ExceptionHelper;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+@QuarkusTestResource(PostgresResource.class) // Needed until there is no test spinning up the PostgresResource in v2 tests.
+class ErrorsAPIV2Test {
```

Comment:
Same as above...

IDK whether it's worth repeating this test; or just move a single implementation to `-core`. WDYT?

It does no harm duplicating it (ideally we could have a super-test-class.. but that may not be worth the effort).

Reply:
Presumably the same applies here.